lib/utils/BaseReporter.js

```diff
@@ -283,6 +343,51 @@ class BaseReporter extends events.EventEmitter {
         }
     }
 
+    getSpecStats (obj) {
+        if (!this.stats.runner[obj.cid]) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
It works basically like a `_.merge` or `_.default``
```js
this.stats.runner[obj.cid] = Object.assign({
    start: new Date(),
    capabilities: obj.capabilities,
    sanitizedCapabilities: sanitize.caps(obj.capabilities),
    config: obj.config,
    specs: {}
}, this.stats.runner[obj.cid]);
```